components/sensor/gdk101.rst

```diff
@@ -0,0 +1,135 @@
+FTLab GDK101 Gamma Radiation Sensor Module
+==========================================
+
+.. seo::
+    :description: Instructions for setting up GDK101 Gamma Radiation Sensor Module
+    :image: gdk101.jpg
+    :keywords: gdk101
+
+The **GDK101** sensor platform allows you to use your GDKA101 sensor
+(`AliExpress`_, `Amazon`_, `DesertCart`_, `RadonShop`_, `FTLab`_) to monitor radiation levels at your place.
```

Comment:
Most of these show out of stock so lets just remove them all.
```suggestion
to monitor radiation levels at your place.
```

Reply:
done